HW9/src/test/java/UserTest.java

```diff
@@ -0,0 +1,120 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserTest {
```

Comment:
Rename all test method according to the convention.  And add tests for exceptional scenarios.